azure-pipelines.yml

```diff
@@ -0,0 +1,17 @@
+name: $(Date:yyyyMMdd).$(Rev:.r)
+pr: 
+- 'master'
+jobs:
+- template: Extension/jobs/build.yml
+  parameters:
+    name: macOS
+    pool:
+      vmImage: 'macOS-10.13'
+
+- template: Extension/jobs/build.yml
+  parameters:
+    name: Linux
+    pool:
+      vmImage: 'ubuntu-16.04'
```

Comment:
Do we want to check Arch or CentOS too or something that is different (aka doesn't support bashisms very well)?

Reply:
Only if you want to maintain the machines for Arch/CentOS